### DISCUSSION TITLE
Integration Home Assistant Core

### DISCUSSION THREAD

**Discussion by Stefano0042** [alandtse/alexa_media_player#1257] [Ideas]
Hello, do you have a plan to extend the Integration component to HA Core?

**Comment by alandtse** [comment]
No. See https://github.com/custom-components/alexa_media_player/discussions/1048

**Comment by mdanilohome** [comment]
Hi, @alandtse ! Can you help me? Alexa Media Player integration is crashing frequently. Scripts created with messages that make up an automation, such as an open door warning, unexpectedly stop responding. I restart Home Assistant and in the integration tab, most of the times the message that it has not been loaded appears in the Alexa integration. To get it back up and running, I am required to delete the integration and attempt to install, after entering the two-step verification code on the Amazon website, it remains indefinitely in a "Upload Alexa Media Player Next Step" screen and does not complete an integration. I continue throughout the day, trying again and again until, in a moment, an integration is effected.
